p5js/NeuralNetwork/NeuralNetwork_ImageClassifier_Letters/sketch.js

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2019 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ml5 Example
+Image classification using MobileNet and p5.js
+This example uses a callback pattern to create the classifier
+=== */
+let nn;
+const IMAGE_WIDTH = 64;
+const IMAGE_HEIGHT = 64;
+const IMAGE_CHANNELS = 4;
+
+let images;
+let testA;
+
+function preload() {
+  images = [];
+  for (let i = 1; i < 7; i++) {
+    const a = loadImage(`images/A_0${i}.png`)
+    const b = loadImage(`images/B_0${i}.png`)
+    images.push({
+      image: a,
+      label: 'a'
+    })
+    images.push({
+      image: b,
+      label: 'b'
+    })
+  }
+
+  testA = loadImage(`images/A_test.png`)
+
+}
+
+function setup() {
+  // load the pixels for each image to get a flat pixel array
+  images.forEach(item => item.image.loadPixels())
+  
+
+  const options = {
+    task: 'imageClassification',
+    debug: true,
+    inputs:[IMAGE_WIDTH, IMAGE_HEIGHT, IMAGE_CHANNELS],
+    layers: [{
+        type: 'conv2d',
+        kernelSize: 5,
+        filters: 8,
+        strides: 1,
+        activation: 'relu',
+        kernelInitializer: 'varianceScaling'
+      },
```

Comment:
For a "starter" example I might suggest eliminating everything but "type" and using defaults (unless there is something specific to this example or that should always be customized)?

Reply:
I will remove the more sophisticated layering from the `_Letters` and `_Letters2` examples in favor of the default specifications for `imageClassificaiton` but will include the layers for the `_Video` example.